src/main/kotlin/no/nav/k9/los/nyoppgavestyring/domeneadaptere/k9klagetillos/K9KlageTilLosAdapterTjeneste.kt

```diff
@@ -149,11 +149,38 @@ class K9KlageTilLosAdapterTjeneste(
             område = "K9",
             kildeområde = "K9",
             type = "k9klage",
-            status = event.aksjonspunkttilstander.lastOrNull()?.status?.kode ?: "OPPR", // TODO statuser må gås opp
+            status = if (event.aksjonspunkttilstander.any { aksjonspunktTilstandDto -> aksjonspunktTilstandDto.status.erÅpentAksjonspunkt() }) {
+                if (oppgaveSkalHaVentestatus(event)) {
+                    "VENTER"
+                } else {
+                    "AAPEN"
+                }
+            } else {
+                "LUKKET"
```

Comment:
Mener å huske at du sa at man helt initelt ikke vil ha aksjonspunkt på behandlingen. Kanskje vi i så fall bør ha en sjekk mot behandlingsstatus for å unngå at behandlingen starter som lukket?